src/rosdep2/platforms/freebsd.py

```diff
@@ -66,7 +66,7 @@ def pkg_info_detect_single(p):
     pop = subprocess.Popen("/usr/sbin/pkg_info -qE " + portname, shell=True)
     return os.waitpid(pop.pid, 0)[1] == 0 # pkg_info -E returns 0 if pkg installed, 1 if not
 
-def pkg_info_detect(packages):
+def pkg_info_detect(packages, warnings=False):
     return [p for p in packages if pkg_info_detect_single(p)]
 
 class PkgAddInstaller(Installer):
```

Comment:
Unrelated to this PR: Is it possible that this is outdated? I think it has to derive from `PackageManagerInstaller`, since it even calls the superclass constructor passing the detect function, with `Installer` doesn't have. I guess there are no freebsd rosdep users then, since AFAICT this cannot work.